cmd/nerdctl/run_linux_test.go

```diff
@@ -104,3 +104,10 @@ func TestRunUlimit(t *testing.T) {
 	base.Cmd("run", "--rm", "--ulimit", ulimit2, testutil.AlpineImage, "sh", "-c", "ulimit -Sn").AssertOutExactly("622\n")
 	base.Cmd("run", "--rm", "--ulimit", ulimit2, testutil.AlpineImage, "sh", "-c", "ulimit -Hn").AssertOutExactly("722\n")
 }
+
+func TestRunWithInit(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	t.Parallel()
+	base := testutil.NewBase(t)
+	base.Cmd("run", "--rm", "--init", "--init-path", "/temp-init/nerdctl-init", testutil.AlpineImage, "ls", "-a").AssertOK()
```

Comment:
can you check that tini is the parent process for the container ? this can be handled with a `ps`

Reply:
Can we just test the time?

```console
$ docker run -d --name without-init alpine sleep infinity
$ time docker stop without-init 
real    0m10.212s
...

$ docker run -d --name with-init --init alpine sleep infinity
$ time docker stop with-init 
real    0m0.191s
...
```

Reply:
I will make a try